### PR TITLE
Minimap

### PR DESCRIPTION
**Pull request by Pommfritz0308**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -44,14 +45,14 @@ OBJS		= $(addprefix $(OBJ_PATH), $(OBJ))
 
 all: $(MLX) $(LIBFT) $(NAME)
 
+$(OBJ_PATH):
```
Comment (wrottger) [inline Makefile:48]: This is part of the makefile branch

File: incl/cub3d.h [hunk 2]
```diff
@@ -39,6 +40,7 @@ typedef struct s_game
 	t_point		step;
 	t_point		s_dist;
 	t_point		tex;
+	int			box_size;
```
Comment (wrottger) [inline incl/cub3d.h:43]: Box size can be a #define

File: src/game_loop.c [hunk 3]
```diff
@@ -61,7 +61,10 @@ static void	dda_helper(t_params *data, int x, t_point ray_dir)
 		perp_wall_dist = (g->s_dist.x - g->d_dist.x);
 	else
 		perp_wall_dist = (g->s_dist.y - g->d_dist.y);
+	if (!perp_wall_dist)
+		perp_wall_dist = 0.1;
 	g->line_height = (int)(WINDOW_HEIGHT / perp_wall_dist);
+	printf("line_height: %d\n", g->line_height);
```
Comment (wrottger) [inline src/game_loop.c:67]: take out the printf

File: src/game_loop.c [hunk 4]
```diff
@@ -155,16 +160,12 @@ void	clear_image(t_params *data)
 	int	i;
 	int	j;
 
-	i = 0;
-	while (i < WINDOW_HEIGHT)
+	i = -1;
```
Comment (wrottger) [inline src/game_loop.c:163]: take out the full function

**Review by wrottger — COMMENTED** [review]
A few possibilities to clean up,  but overall good work

**Review by Pommfritz0308 — COMMENTED** [review]
bruh

**Review by wrottger — APPROVED** [review]
Excellen work

**Pull request merged**